README.md

```diff
@@ -1,57 +1,74 @@
+  * `http://[project].local:8000` if you're running Butler on a Vagrant
+  * [http://localhost:8000](http://localhost:8000) if you're developing locally
 
-````
-"scripts": {
-	"butler": "gulp --gulpfile node_modules/butler/gulpfile.js develop",
-	"linting": "gulp --gulpfile node_modules/butler/gulpfile.js test",
-	"deploy": "gulp --gulpfile node_modules/butler/gulpfile.js deploy"
-},
-````
+## What does this Butler do?
 
-1.  Run `npm run butler` and begin developing normally
+* `npm run butler`
 
-1.  Navigate to `localhost:8000` or `[project].local:8000` in your browser to see the styleguide
+  This is the default task. This will watch your sass/sculpin files for changes and compile/build accordingly. It will also flag any sass linting errors before compiling. It will output CSS that has been been minified and optimized.
 
-1.  Add `node_modules` to your project's `.gitignore` file and commit your project's new `package.json`
+* `npm run linting`
 
-## What does this Butler do
-*  `npm run butler` 
-    
-    This is the default task. This will watch your sass/sculpin files for changes and compile/build accordingly. It will also flag any sass linting errors before compiling. It will output CSS that has been been minified and optimized. 
+  This is the testing task it will run linters as their own tasks. To learn more about configuring and customizing the linters for Butler check the [linters documentation](/docs/LINTERS.md).
 
-*  `npm run linting`
+* `npm run deploy`
 
-    This is the testing task it will run linters as their own tasks. To learn more about configuring and customizing the linters for Butler check the [linters documentation](/docs/LINTERS.md).
+  This is a task to deploy the static styleguide to GitHub pages.
 
-*   `npm run deploy`
+  Butler will build a Sculpin production artifact to `styleguide/output_prod` and deploy the production artifact to `gh-pages` branch of the repo defined in the `conf/butler.defaults.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]".
 
-    This is a task to deploy the static styleguide to GitHub pages. It will deploy to `gh-pages` branch of the repo defined in the `config/paths.js`. Each commit for this process will default to the message: "Updated with Butler - [timestamp]". *Note: When you are deploying, Butler will ask you for your github credentials multiple times. Enter your own github credentials as prompted.*
+  You may want to create a `sculpin_site_prod.yml` to define the site URL once deployed. You can find out more information about environment aware configuration for Sculpin [here](https://sculpin.io/documentation/configuration/).
+
+  *Note: When you are deploying, Butler will ask you for your GitHub credentials at least once, possibly multiple times. Enter your own GitHub credentials as prompted.*
 
 ## Making Changes to Butler
+
 Please feel free to change/extend/break this Gulpfile to fit the specific needs of the project.
```

Comment:
This is one thing I'm not sure about. Since the Butler Gulpfile isn't checked into the project, how would one change/extend/break it for the project?

Potential answers:
* Create branches and PRs in the Butler repo
* Copy the Gulpfile to... somewhere in your project... and update the `scripts` in your project's `packages.json` file